srcpkgs/mtpaint/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'mtpaint'
 pkgname=mtpaint
-version=3.49.33
+version=3.49.34
 revision=1
-_commit=5272e2b1e773c8e02ac3506b2d3bde82ad946b21
+_commit=b30486e0594d7ac3bf8d4242d74d910d3172a9b9
 wrksrc="mtPaint-${_commit}"
 build_style=configure
 configure_args="--prefix=/usr --mandir=/usr/share/man
-		imagick cflags GIF jpeg jp2 tiff lcms2 man
+		imagick cflags GIF jpeg jp2v2 tiff lcms2 man
 		gtk3 webp intl release"
```

Comment:
I think this should be indented with spaces, right? Could you fix it?

Reply:
Οκ 👍

Reply:
Thank you, done :)